pkgs/development/python-modules/mistune/default.nix

```
@@ -27,6 +27,7 @@ buildPythonPackage rec {
   pythonImportsCheck = [ "mistune" ];
 
   meta = with lib; {
+    changelog = "https://github.com/lepture/mistune/blob/${src.rev}/docs/changes.rst";
```

```suggestion
    changelog = "https://github.com/lepture/mistune/blob/${version}/docs/changes.rst";
```

There is no `src.rev`, since you are using fetchPypi.